backend_API/CONTRIBUTING.md

```diff
@@ -0,0 +1,176 @@
+# Contributing Guidelines:
+
+We welcome all contributions to the Open Science Community in Saudi Arabia. The contribution can be an [issue report](https://github.com/Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi/issues)
+or a [pull request](https://github.com/Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi/pulls).
+
+## BACKEND API for MOOCs platform
+
+This project is meant to serve as the backend API for the MOOCs platform. The program aims to build an educational platform to serve courses and videos, where each course may have exercises attached.
+
+The API is hosted for testing at https://moocs-test.onrender.com/
+
+#### API DOCUMENTATION
+Documentation for consuming the API endpoints can be found here https://documenter.getpostman.com/view/20633788/2s8YemuZwv
+
+#### Technologies
+
+- Node JS - Backend Environment
```

Comment:
```suggestion
- [Node JS](https://nodejs.org/en/) - Backend Environment
```
Can you add links for Express JS, Mongo DB, Mocha and Chai, please?

Reply:
Alright @BatoolMM